File: Capitulo_2/Python/C_Exercicios_Propostos(pag-50-ate-53)/exercicio-6.py
from math import sqrt

a = float(input('Informe o valor de A:'))
b = float(input('Informe o valor de B:'))
c = float(input('Informe o valor de C:'))

resultado_positivo = (-b + sqrt(b**2 - 4*a*c))/2*a
resultado_negativo = (-b - sqrt(b**2 - 4*a*c))/2*a

print(f'Equação de segundo grau positiva:{resultado_positivo}')
print(f'Equação de segundo grau negativo:{resultado_negativo}')

# Essa questão está faltando um pedaço, logo é inconclusivo se está correto.

File: Capitulo_2/JavaScript/C-Exercicios-Propostos(pag-50-ate-53)/exercicio-6.js
const prompt = require('prompt-sync')()

let a = Number(prompt('Informe o valor de A:'))
let b = Number(prompt('Informe o valor de B:'))
let c = Number(prompt('Informe o valor de C:'))

let resultadoPositivo = (-b + Math.sqrt(b**2 -4*a*c))/2*a
let resultadoNegativo = (-b - Math.sqrt(b**2 -4*a*c))/2*a

console.log(`Equação de segundo grau positiva:${resultadoPositivo}`)
console.log(`Equação de segundo grau negativo:${resultadoNegativo}`)

